index.html

```diff
@@ -16,10 +16,16 @@
   <body>
     <header>
       <div id="logo">Mr. Mandal</div>
+      <div id="overlay">
+        <img id="close-button" src="./images/icons/close-icon.svg" alt="close button">
+        <a  class="menu-link" href="#projects">Portoflio</a>
+        <a  class="menu-link" href="#about-myself">About</a>
+        <a class="menu-link" href="#contact-form">Contact</a>
+      </div>
```

Comment:
Kindly, fix the background to be `blur` and `font-weight` of the menu option when the menu options appear. You can use something like this `filter: blur(..px)` or you can use anything that can work for you to achieve the last result as shown here in the Figma design:

| Your Design | Figma Design |
|-------------|---------------|
| <img width="366" alt="image" src="https://user-images.githubusercontent.com/34578631/167387056-3c127ff7-2e6a-4984-b837-7cab52ac8aa6.png"> | <img width="366" alt="image" src="https://user-images.githubusercontent.com/34578631/167386951-a3dbe475-e9ef-4f2c-8f54-dc243305db43.png">

Reply:
Thank you for your feedback. I have made the required changes. @Mre55